src/App.tsx

```diff
@@ -80,6 +81,13 @@ function App() {
     type: 'FeatureCollection',
     features: []
   });
+  // Campus Marker
+  interface CampusCoordination {
+    CampCordLng: number;
+    CampCordLat: number;
+  }
```

Comment:
This doesn't need to be changed right now, but I'm going to move this interface definition out of this file and put it someplace else. This main file is starting to get really crowded and we need to try to clean it up a little bit.